icu4c/source/common/uresbund.cpp

```diff
@@ -1245,6 +1246,7 @@ static UResourceBundle *init_resb_result(const ResourceData *rdata, Resource r,
     /*resB->fParent = parent->fRes;*/
     uprv_memmove(&resB->fResData, rdata, sizeof(ResourceData));
     resB->fSize = res_countArrayItems(&(resB->fResData), resB->fRes);
+    ResourceTracer(resB).traceOptional("get");
```

Comment:
Why is this optional? resB is not null, or else the previous line crashed. So this will always trace, won't it?